src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ActionPlanService.kt

```diff
@@ -91,10 +91,12 @@ class ActionPlanService(
 
   fun approveActionPlan(id: UUID, user: AuthUser): ActionPlan {
     val actionPlan = getActionPlan(id)
+
+    deliverySessionService.createUnscheduledSessionsForActionPlan(actionPlan)
```

Comment:
should we move this below `actionPlanRepository.save(actionPlan)` so that checks on already approved plans can prevent duplicate `createUnscheduledSessionsForActionPlan`?